### DISCUSSION TITLE
Lab Assignment #1 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#2] [Assignments]
## Due 4/13 before class 

Please use this thread to post your [lab assignment's #1](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week1/lab_assignment/) link, which is the GitHub pages URL.

For example:
https://albertkun.github.io/23s-asiaam-191a-sample/week1/index.html

**Comment by zacharypakin** [comment]
https://zacharypakin.github.io/aa191a-23s/week01/index.html

**Comment by czhangy** [comment]
https://czhangy.github.io/ASIA-AM-191A/week1/index.html

**Comment by SageLuong** [comment]
https://sageluong.github.io/23s-asianam-191a-labs/week1/index.html

**Comment by qatle** [comment]
https://trinhqale.github.io/AM191A-labs/week1/index.html

**Comment by kmtsujimoto** [comment]
https://kmtsujimoto.github.io/23s-asiaam-191a-labs/week01/index.html

**Comment by vickydelk** [comment]
https://vickydelk.github.io/aa191-assignments/week1/index.html

**Comment by cassidytu** [comment]
[https://cassidytu.github.io/asam191a-labs/week%201/index.html](https://cassidytu.github.io/asam191a-labs/week%201/index.html)

**Comment by catherinehhu** [comment]
https://catherinehhu.github.io/191a-labs/week1/index.html

**Comment by jamesfalkenroth** [comment]
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week1/index.html

**Comment by shintuzd** [comment]
https://shintuzd.github.io/23s-asiaam-191a-assignments/week1/index.html

**Comment by paige-lee** [comment]
https://paige-lee.github.io/aa191a-labs/week1/index.html

**Comment by maizaha** [comment]
https://maizaha.github.io/23s-asiaam-191a-labs/week1/index.html

**Comment by stellakangj** [comment]
https://stellakangj.github.io/aa191a/week1/

**Comment by danigold1020** [comment]
https://danigold1020.github.io/23s-asianam-191a-labs/week01/index.html

**Comment by cassiuscarandang** [comment]
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week1/

**Comment by chrysanthemma** [comment]
https://chrysanthemma.github.io/asainam191a-labs/week1/index.html

**Comment by DylanJWan** [comment]
https://dylanjwan.github.io/asiaam-191a-assignments/week1/

**Reply by albertkun** [reply]
@DylanJWan the url above is a local one, you may want to check this part of the lab:

https://albertkun.github.io/23S-ASIAAM-191A/labs/week1/3/#using-github-pages

**Comment by angelicaalacantar** [comment]
https://angelicaalacantar.github.io/aas191a-23s/week01/index.html

**Comment by JonathanGLopez** [comment]
https://jonathanglopez.github.io/Asiaam191aLB1/

**Comment by joanneluong** [comment]
https://joanneluong.github.io/aas191a-labs/week1/index.html

**Comment by amandatlxu** [comment]
https://amandatlxu.github.io/

**Comment by ariscorpus** [comment]
https://ariscorpus.github.io/23s-asianam-191a-labs/week1

**Comment by mandymwlai** [comment]
https://mandymwlai.github.io/AA-191-lab/week1/

**Comment by emai14** [comment]
https://emai14.github.io/23S-ASAM191A-Labs/week01/

**Comment by rebeccaksuen** [comment]
https://rebeccaksuen.github.io/23s-asiaam-191a-labs/Week%201/

**Comment by trufflelover** [comment]
https://trufflelover.github.io/23s-asiaam-191a-labs/week1/

**Comment by JonathanGLopez** [comment]
https://jonathanglopez.github.io/Asiaam191aLB1/week1/

**Comment by enyi-emuka** [comment]
https://enyi-emuka.github.io/asiaam191a_labs/week1/index.html

**Comment by alanamalika** [comment]
https://alanamalika.github.io/23s-asianam-191a-labs/week01/index.html
